tests/queries/0_stateless/02160_client_autocomplete_parse_query.expect

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/expect -f
+
+log_user 0
+set timeout 60
+set uuid ""
+match_max 100000
+expect_after {
+    # Do not ignore eof from read.
+    eof { exp_continue }
+    # A default timeout action is to do nothing, change it to fail
+    timeout { exit 1 }
+}
+
+set basedir [file dirname $argv0]
+spawn bash -c "source $basedir/../shell_config.sh ; \$CLICKHOUSE_CLIENT_BINARY \$CLICKHOUSE_CLIENT_OPT"
+expect ":) "
+
+# Make a query
+send -- "set max_distributed"
+expect "set max_distributed"
+
+# Wait for suggestions to load, they are loaded in background
+set is_done 0
+set timeout 1
+while {$is_done == 0} {
+    send -- "\t"
+    expect {
+        "_" {
+            set is_done 1
+        }
+        default {
+            # Reset the expect_after
+        }
+    }
+}
+set timeout 60
+send -- ""
```

Comment:
What is the character?

Reply:
Ctrl-C

Reply:
Can we escape it here somehow, or at least write a comment about that?

Reply:
Sure, done.